docker/service/src/main/java/uk/nhs/adaptors/scr/config/ExceptionHandlerFilter.java

```diff
@@ -0,0 +1,49 @@
+package uk.nhs.adaptors.scr.config;
+
+import static uk.nhs.adaptors.scr.controllers.FhirMediaTypes.APPLICATION_FHIR_JSON;
+import static uk.nhs.adaptors.scr.controllers.FhirMediaTypes.APPLICATION_FHIR_JSON_VALUE;
+
+import java.io.IOException;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.OperationOutcome;
+import org.springframework.http.HttpStatus;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import uk.nhs.adaptors.scr.components.FhirParser;
+import uk.nhs.adaptors.scr.exceptions.NHSCodings;
+
+public class ExceptionHandlerFilter extends OncePerRequestFilter {
+
+    private static final String HEADER_NAME = "X-Correlation-ID";
+    private static final String UUID_REGEX = "^[0-9a-fA-F]{8}+-[0-9a-fA-F]{4}+-[0-9a-fA-F]{4}+-[0-9a-fA-F]{4}+-[0-9a-fA-F]{12}+$";
+
+    @Override
+    public void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws ServletException,
+        IOException {
```

Comment:
Not a big thing but try to keep same signature parts on the same line. Move "throws..." to new line, or IOException to previous